kodi_addon_checker/check_files.py

```diff
@@ -90,6 +91,19 @@ def addon_xml_matches_folder(report: Report, addon_path: str, parsed_xml, folder
             report.add(Record(PROBLEM, "Addon id and folder name does not match."))
 
 
+def check_for_legacy_changelog(report: Report, addon_path: str, branch: str):
+    """Check whehter a legacy changelog.txt is present in the addon folder root"""
+    changelog_path = os.path.join(addon_path, "changelog.txt")
+    if os.path.isfile(changelog_path):
+        report.add(
+            Record(
+                PROBLEM if KodiVersion(branch) >= KodiVersion("leia") else WARNING,
```

Comment:
Probably not a good idea to report it as problem. Maybe turn this into a warning only f the news is empty? cc @ronie

Reply:
well, if you can point me to 1 online article that says it is not a good idea to accompany your software with a changelog file, then i'm all ears ;-)